docs/telemetry.md

```diff
@@ -0,0 +1,20 @@
+---
+sidebar_label: Telemetry
+title: Telemetry
+---
+
+Apify collects telemetry data about general usage of CLI to help us improve the product.
+Participation in this program is optional, and you may opt-out if you'd not like to share any information.
+
+## How is the data collected
+
+We currently collect the following metrics:
+
+- Usage of commands except `secrets` commands.
+- Templates used for `actor create` command.
+
+All telemetry data are collected and stored securely on Mixpanel.
+
+## How to opt-out
+
+To opt out of telemetry, set the `APIFY_CLI_TELEMETRY_DISABLE` environment variable to 1 or true.
```

Comment:
`APIFY_CLI_DISABLE_TELEMETRY` may sound a bit better. 

Plus I'd just say one or another. There is no point in giving 2 options. Just unneeded extra information.